items/swords/Voidforge.yml

```diff
@@ -0,0 +1,18 @@
+acquisitionMethods:
+  - type: plainText
+    text: "'The Endless Darkness' divination cards can be passively farmed while you target-farm Harbinger uniques from the boss encounter in T15 Beachhead maps."
+    referenceLink: https://poe.fish/fishing-items/anglers-plait
+  - type: boss
+    boss: Uber Elder
+    uber: true
+    frequency: extremely-rare
+    wiki: https://www.poewiki.net/wiki/The_Elder_(The_Shaper%27s_Realm)#Drops
+  - type: typicalDivCardFarm
+    investment: high
```

Comment:
On one hand it's plausible that some players will acquire Voidforge by passively accruing cards (and/or harvest-gambling them) way before they get one as a drop, but on the other hand the ability to target-farm it could be deceptive (when implied by this section) in the sense that you can't juice the content listed here, you can only just try to get more attempts at it.

But I agree that this is an important avenue of acquisition to keep in mind, maybe we need to better revise the "investment" field's meaning in a wider context and perhaps add a value for "cannot be influenced" which describes situations like this. Happy to merge with this section as-is though!